miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/ApiClient.kt

```diff
@@ -101,13 +106,36 @@ internal class RetrofitRequestExecutor(
     private val retrofit: Retrofit
 ) {
 
-    private inline fun <reified T : ErrorResponse> createErrorConvertor(retrofit: Retrofit) =
+    private var retryCount = 0
```

Comment:
This can't be a class property because it will mean that the property is shared by ALL network requests, but we want it to be specific to EACH network reqeust. Also, currently any subsequent network requests will already have `retryCount = 5`, so the network request won't even be attempted.

I think the retry count should be passed to the `executeRequest` method and give it a default value of 0. See the iOS implementation: https://github.com/rakutentech/ios-miniapp/commit/42d373a9c69ef2ab4890ea58f03a57ea9a4c8b27

Reply:
Yeah, I have updated this part.